iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/DeviceRegistryExample.java

```diff
@@ -93,15 +93,16 @@ public static Topic createIotTopic(String projectId, String topicId) throws Exce
 
     try (TopicAdminClient topicAdminClient = TopicAdminClient.create()) {
       final Topic topic = topicAdminClient.createTopic(topicName);
-      Policy policy = topicAdminClient.getIamPolicy(topicName.toString());
+      com.google.iam.v1.Policy policy = topicAdminClient.getIamPolicy(topicName.toString());
       // add role -> members binding
       Binding binding =
           Binding.newBuilder()
               .addMembers("serviceAccount:cloud-iot@system.gserviceaccount.com")
               .setRole(Role.owner().toString())
               .build();
       // create updated policy
-      Policy updatedPolicy = Policy.newBuilder(policy).addBindings(binding).build();
+      com.google.iam.v1.Policy updatedPolicy =
```

Comment:
also, not a fan.

Reply:
This is the code that does not appear in documentation, it's necessary for testing the API and is a convenient helper if someone is using this to automate device provisioning.